File: feature_movie/src/main/java/java/io/nanodegree/popularmovies/feature/movie/data/retrofit/response/MovieContainer.java
package java.io.nanodegree.popularmovies.feature.movie.data.retrofit.response;

import com.squareup.moshi.Json;

import java.io.nanodegree.popularmovies.feature.movie.data.model.Movie;
import java.util.List;

/**
 * Created by Andrea Loddo (andrea@evenly.io) on 20.12.2018
 * for Evenly GmbH
 * <p>
 * Copyright (c) 2018 Evenly GmbH,
 * all rights reserved
 */
public class MovieContainer {

    @Json(name = "results")
    public List<Movie> movieList;
}

File: feature_movie/src/main/java/java/io/nanodegree/popularmovies/feature/movie/data/retrofit/response/MovieContainer.kt
package java.io.nanodegree.popularmovies.feature.movie.data.retrofit.response

import com.google.gson.annotations.SerializedName

import java.io.nanodegree.popularmovies.feature.movie.data.model.Movie

/**
 * Created by Andrea Loddo (andrea@evenly.io) on 20.12.2018
 * for Evenly GmbH
 *
 *
 * Copyright (c) 2018 Evenly GmbH,
 * all rights reserved
 */
data class MovieContainer (@field:SerializedName("results") val movieList: List<Movie>?)

